primitive/crypto/src/dress/curve.rs

```diff
@@ -47,10 +63,9 @@ macro_rules! curve_operation {
             const PARAM_B: $field = $b;
 
             fn to_affine(self) -> Self::Affine {
-                let inv_z = self.z.invert().unwrap();
                 Self::Affine {
-                    x: self.x * inv_z,
-                    y: self.y * inv_z,
+                    x: self.x / self.z,
+                    y: self.y / self.z,
```

Comment:
From calculation amount perspective, we should get invert and multiply by x and y.
The division perform invert for each so we can reduce invert times.